base/show.jl

```diff
@@ -44,6 +44,11 @@ function show(io::IO, ::MIME"text/plain", f::Function)
     end
 end
 
+show(io::IO, ::MIME"text/plain", f::Fix1) = print(io, f.f, "(", f.x, ") (fix argument 1)")
+show(io::IO, ::MIME"text/plain", f::Fix2) = print(io, f.f, "(", f.x, ") (fix arguemnt 2)")
```

Comment:
```suggestion
show(io::IO, ::MIME"text/plain", f::Fix2) = print(io, f.f, "(", f.x, ") (fix argument 2)")
```

Reply:
Sorry this suggestion doesn't change anything?

Reply:
`arguemnt` -> `argument` ? it's just a typo